libr/core/cmd_open.c

```diff
@@ -285,6 +286,7 @@ static void cmd_open_bin(RCore *core, const char *input) {
 					r_bin_open_io (core->bin, &opt);
 					r_io_desc_close (desc);
 					r_core_cmd0 (core, ".is*");
+					r_io_use_fd(core->io, saved_fd);
```

Comment:
Same here. Also did u checked if issue happens too in the else filename branch?

Reply:
no issue when using `oba 0x804d7000`